auctions/models.py

```diff
@@ -573,13 +578,11 @@ def __unicode__(self):
         return "%s credit for %s" % (self.fee_type, self.payout)
 
 
-@receiver(post_save, sender=Payout)
-@receiver(post_save, sender=Offer)
 @receiver(post_save, sender=Bid)
 @receiver(post_save, sender=Claim)
 @receiver(post_save, sender=Vote)
 def update_datetimes_for_model_save(sender, instance, created, **kwargs):
     if created:
-        sender.objects.filter(id=instance.id).update(created=datetime.now())
+        sender.objects.filter(id=instance.id).update(created=timezone.now())
     else:
-        sender.objects.filter(id=instance.id).update(modified=datetime.now())
+        sender.objects.filter(id=instance.id).update(modified=timezone.now())
```

Comment:
There's another use of `datetime.now()` in `notify_matching_askers`. I'll file an issue to replace all (or as many as possible) uses of `datetime.now()` with `timezone.now()`.